runtime/gc_glue_java/ArrayletObjectModel.hpp

```diff
@@ -923,6 +923,36 @@ class GC_ArrayletObjectModel : public GC_ArrayletObjectModelBase
 		*dataAddrPtr = calculatedDataAddr;
 	}
 
+	/**
+	 * Asserts that an indexable object pointer is indeed an indexable object
+	 *
+	 * @param arrayPtr      Pointer to the indexable object
+	 */
+	void AssertArrayPtrIsIndexable(J9IndexableObject *arrayPtr);
+
+	/**
+	 * Sets data pointer of indexable objects.
+	 * Sets the data pointer of either a contiguous or discomtiguous indexable
+	 * object.
+	 *
+	 * @param arrayPtr      Pointer to the indexable object whose size is required
+	 */
+	MMINLINE void
+	setObjectDataAddr(omrobjectptr_t arrayPtr)
```

Comment:
now that is lost addr parameter, it could be 'fixup; (again, not that I like it over 'update', but it's just already widespread)